cypress/support/commands.js

```diff
@@ -9,7 +9,7 @@ Cypress.Commands.add('runAndAwait', (callable, method='GET', addArtificialWait=f
     cy.wait(`@${requestId}`);
     cy.route(method, '**').as('untrackedRequest'); // stop recording requests
 
-    if (addArtificialWait) cy.wait(5000);
+    if (addArtificialWait) cy.wait(8000);
```

Comment:
@rakesh-1010 why are we increasing the time here ??

Reply:
e2e tests were failing because of timeout issue. It passed with increasing the time. But this should not be any solution. We should keep a separate card to work on the e2e tests.